plugins/Web/test.py

```diff
@@ -179,6 +179,31 @@ def testWhitelist(self):
                 conf.supybot.plugins.Web.urlWhitelist.set('')
                 conf.supybot.plugins.Web.fetch.maximum.set(fm)
 
+        def testtitleOembedRegistry(self):
+            try:
+                conf.supybot.plugins.Web.useOembedRegistry.setValue(True)
+                self.assertResponse(
+                    'title https://www.flickr.com/photos/bees/2362225867/',
+                    'Bacon Lollys')
+            finally:
+                conf.supybot.plugins.Web.useOembedRegistry.setValue(False)
+
+        def testtitleOembedDiscovery(self):
+            try:
+                conf.supybot.plugins.Web.useOembedDiscovery.setValue(True)
+                self.assertResponse(
+                    'title https://flickr.com/photos/bees/2362225867/',
+                    'Bacon Lollys')
+            finally:
+                conf.supybot.plugins.Web.useOembedDiscovery.setValue(False)
+
+        def testtitleOembedError(self):
+            try:
+                conf.supybot.plugins.Web.useOembedDiscovery.setValue(True)
+                self.assertError('title https://nonexistent.example.com/post/123')
+            finally:
+                conf.supybot.plugins.Web.useOembedDiscovery.setValue(False)
```

Comment:
```suggestion
        def testtitleOembedRegistry(self):
            with conf.supybot.plugins.Web.useOembedRegistry.context(True):
                self.assertResponse(
                    'title https://www.flickr.com/photos/bees/2362225867/',
                    'Bacon Lollys')

        def testtitleOembedDiscovery(self):
            with conf.supybot.plugins.Web.useOembedDiscovery.context(True):
                self.assertResponse(
                    'title https://flickr.com/photos/bees/2362225867/',
                    'Bacon Lollys')

        def testtitleOembedError(self):
            with conf.supybot.plugins.Web.useOembedDiscovery.context(True):
                self.assertError('title https://nonexistent.example.com/post/123')
```

use the new style, it's shorter